src/main/java/gregtech/api/capability/impl/AbstractRecipeLogic.java

```diff
@@ -646,7 +646,7 @@ protected void setupRecipe(Recipe recipe) {
         this.fluidOutputs = GTUtility.copyFluidList(recipe.getAllFluidOutputs(metaTileEntity.getFluidOutputLimit()));
         this.itemOutputs = GTUtility.copyStackList(recipe.getResultItemOutputs(
                 GTUtility.getTierByVoltage(recipe.getEUt()),
-                getOverclockTier(),
+                getOverclockForTier(getEnergyCapacity()),
```

Comment:
Should this be using `getOverclockForTier(getMaximumOverclockVoltage)`? For Multiblocks that would make it use `MaxVoltage` instead of its Energy Capacity, which I think is more in line with the tiers for multiblocks. And for single blocks, `getMaximumOverclockVoltage` takes into account the tier being limited by the overclock button